templates/common/on-prem/files/keepalived.yaml

```
@@ -100,8 +100,8 @@ contents:
           remove_vip()
           {
             address=$1
-            interface=$(ip -o a | grep '\s${address}/' | awk '{print $2}')
-            cidr=$(ip -o a | grep '\s${address}/' | awk '{print $4}')
+            interface=$(ip -o a | grep "\s${address}/" | awk '{print $2}')
+            cidr=$(ip -o a | grep "\s${address}/" | awk '{print $4}')
```

Nit: I'd personally remove the grep in the middle and rely on awk for filtering:
```suggestion
            interface=$(ip -o a | awk "/\s${address}/ {print \$2}")
            cidr=$(ip -o a | awk "/\s${address}/ {print \$4}")
```